inspectors/models/notation/sectionbreaks/sectionbreaksettingsmodel.cpp

```diff
@@ -0,0 +1,62 @@
+#include "sectionbreaksettingsmodel.h"
+
+SectionBreakSettingsModel::SectionBreakSettingsModel(QObject* parent, IElementRepositoryService* repository) :
+    AbstractInspectorModel(parent, repository)
+{
+    setModelType(TYPE_SECTIONBREAK);
+    setTitle(tr("Section Break"));
+    createProperties();
+}
+
+void SectionBreakSettingsModel::createProperties()
+{
+    m_startWithLongInstrNames = buildPropertyItem(Ms::Pid::START_WITH_LONG_NAMES, [this] (const int pid, const QVariant& newValue) {
+        onPropertyValueChanged(static_cast<Ms::Pid>(pid), newValue);
+
+        emit requestReloadPropertyItems();
+    });
+
+    m_resetBarNums = buildPropertyItem(Ms::Pid::START_WITH_MEASURE_ONE, [this] (const int pid, const QVariant& newValue) {
+        onPropertyValueChanged(static_cast<Ms::Pid>(pid), newValue);
+
+        emit requestReloadPropertyItems();
+    });
+
+    m_pause = buildPropertyItem(Ms::Pid::PAUSE);
```

Comment:
Hmmm, why are the enlengthened `buildPropertyItem()` necessary for the first two properties?

Reply:
See above